app/assets/javascripts/rules/story.js

```diff
@@ -29,12 +29,12 @@ export const classIconRule = (story_type) => {
 };
 
 export const isStoryDontEstimated = (story_type, estimate) => (story_type === 'feature' && !estimate);
+export const IsRelease = (story_type) => (story_type === 'release');
```

Comment:
You can use `isRelease` insted of `IsRelease`, because it is not a react component

Reply:
Ok!